plugins/ComposeAuth/src/androidMain/kotlin/io/github/jan/supabase/composeAuth/Utils.kt

```diff
@@ -0,0 +1,20 @@
+package io.github.jan.supabase.composeAuth
```

Comment:
Although its allowed as the [docs](https://kotlinlang.org/docs/coding-conventions.html#naming-rules) say. We can separate so that in the future we can have `compose.db` etc.